documentation/frontend/reference/testing_guidelines.md

```diff
@@ -357,10 +357,29 @@ output of failed tests by finding the `trace.zip` for the relevant test in the
 [Playwright Trace Viewer](https://playwright.dev/docs/trace-viewer) to inspect
 these (or open the zip yourself and poke around the files on your own).
 
-For Playwright tests failing in CI, a GitHub comment will appear with a link to
-download an artifact of the `test-results` folder.
+Playwright tests in CI are run with `-u` option by default, this means that
+snapshots will automatically be updated for modified parts of the UI if
+Playwright detects that. See
+[Updating snapshots](/frontend/guides/test.md#updating-snapshots) for more
+reading about this.
+
+When this happens, a GitHub comment will appear with a link to download
+artifacts named in the form `*_snapshot_diffs`. The resulting artifacts would
+each contain a zipped patch of the snapshot changes named
+`snapshot_diff.patch.gz`, once downloaded, they can be unzipped and applied to
+your working branch by running:
+
+`gunzip -k -c snapshot_diff.patch.gz | git apply -`
+
+```{note}
+It is assumed that you'd already have `gunzip` installed as it looks to be available on macOS and is usually available on most Linux distros. If this is not your case, you might need to have that installed first.
+```
```

Comment:
Good note. It is available in `ov`, so the instructions can be `cat snapshot_diff.patch.gz | ov gunzip -kc - | git apply`. gzip is usually available, you're right, but we can do away with the question altogether using `ov` :slightly_smiling_face: